messaging/src/main/java/org/axonframework/eventhandling/pooled/Coordinator.java

```diff
@@ -67,6 +67,7 @@
     private final Consumer<? super TrackedEventMessage<?>> ignoredMessageHandler;
     private final BiConsumer<Integer, UnaryOperator<TrackerStatus>> processingStatusUpdater;
     private final long tokenClaimInterval;
+    private final long tokenExtensionThreshold;
```

Comment:
Could we call it the `claimExtensionThreshold` here, too, so as not to confuse ourselves with the variable in the `PooledStreamingEventProcessor` and the `WorkPackage`?